doc/whatsnew/fragments/7311.false_negative

```diff
@@ -0,0 +1 @@
+fix `undefined-loop-variable` with `break` and `continue` statements in `else` blocks
```

Comment:
```suggestion
Fix `undefined-loop-variable` with `break` and `continue` statements in `else` blocks

Refs #7311 
```

Reply:
i applied this change but the newsfragments check is still failing, no idea why

Reply:
I had to investigate for an humiliatingly long time, it seems the ``.`` is required. Regexes are hard 😄